tests/lib/ocpp/v201/test_smart_charging_handler.cpp

```diff
@@ -916,4 +922,73 @@ TEST_F(ChargepointTestFixtureV201, K01_ValidateProfile_IfChargeStationMaxProfile
     EXPECT_THAT(sut, testing::Eq(ProfileValidationResultEnum::Valid));
 }
 
+TEST_F(ChargepointTestFixtureV201,
+       K01FR14_IfTxDefaultProfileWithSameStackLevelDoesNotExist_ThenApplyStationWideTxDefaultProfileToAllEvses) {
+
+    auto profile = create_charging_profile(DEFAULT_PROFILE_ID, ChargingProfilePurposeEnum::TxDefaultProfile,
+                                           create_charge_schedule(ChargingRateUnitEnum::A), uuid(),
+                                           ChargingProfileKindEnum::Absolute, DEFAULT_STACK_LEVEL);
+
+    auto sut = handler.add_profile(STATION_WIDE_ID, profile);
+
+    EXPECT_THAT(sut.status, testing::Eq(ChargingProfileStatusEnum::Accepted));
+    EXPECT_THAT(handler.get_profiles(), testing::Contains(profile));
```

Comment:
Building on that last comment about `get_profiles()`, I don't see the point in testing it like this. From the test names they seem like good tests but I expect this to be tested using some kind of composite results in the future since that should be the primary interface of this.

Reply:
I'm unclear if you are saying that you want the composite results/future for this PR? For now, we have a straightforward and simple way to verify that profiles are being added, and if this behavior can be tested a different way in the future, we can iterate/incrementally build to that state.

Reply:
I don't expect any changes based on this comment. This was just elaborating on my other comment about exposing the `get_profiles()` function. I agree that we can iterate this in the future once we have some more composite schedule functions.